src/tests/unit/tests/injected/components/details-dialog.test.tsx

```diff
@@ -156,18 +187,13 @@ describe('DetailsDialog', () => {
                     deps,
                     failedRules: expectedFailedRules,
                     dialogHandler: dialogDetailsHandlerMockObject,
-                };
+                } as any;
 
-                const wrapper = shallow<DetailsDialog>(<DetailsDialog {...props} />);
-
-                const commandBar = wrapper.find(CommandBar);
-
-                commandBar.prop('onClickCopyIssueDetailsButton')(eventStub);
-
-                clickHandlerMock.verify(
-                    handler => handler(wrapper.instance(), eventStub),
-                    Times.once(),
+                render(<DetailsDialog {...props} />);
+                getMockComponentClassPropsForCall(CommandBar).onClickCopyIssueDetailsButton(
+                    eventStub,
                 );
+                clickHandlerMock.verify(handler => handler(It.isAny(), eventStub), Times.once());
```

Comment:
We can get closer to the original `clickHandlerMock.verify` call (verifying that the component is passed in) by doing the following:

```js
const testSubject = new DetailsDialog(props);
render(testSubject.render());
.
.
.
clickHandlerMock.verify(handler => handler(testSubject, eventStub), Times.once());
```